doc/getting_started.rst

```diff
@@ -235,7 +235,41 @@ Now you can use **Lua Fun**:
     2 c
     one
 
-Please note that functions support multireturn.
+Please note that functions support multireturn. It means that some of them may
+return more than one iterator, for example functions ``partition`` and ``span``.
+
+One call ``:unwrap()`` on generators to pass them into the ``for <..> in <...>``
+loop. Example:
+
+.. code-block:: bash
+
+   it1, it2 = fun.range(10):partition(function(i) return i % 2 == 0 end)
+   for i in it1:unwrap() do print(i) end
```

Comment:
Can we add angle brackets as prompt to mark input lines? As I see, it is used above.

Reply:
Sure, added.